CHANGELOG.asciidoc

```diff
@@ -25,10 +25,26 @@ Notes:
         ==== x.x.x - YYYY/MM/DD
 ////
 
-
 [[release-notes-3.x]]
 === Node.js Agent version 3.x
 
+==== Unreleased
+
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+* feat: implemented sanitize_feature_names specification
+
+        Allows users to configure a list of wildcard patterns to _remove_ items
+        from the agent's HTTP header and `application/x-www-form-urlencoded` payloads.
+
+        spec: https://github.com/elastic/apm/blob/master/specs/agents/sanitization.md
+        docs: https://github.com/elastic/apm-agent-nodejs/blob/master/docs/configuration.asciidoc#sanitize-field-names
```

Comment:
Spacing like this doesn't render well in asciidoc. For example, our build  assumes these are undefined code blocks:
<img width="758" alt="Screen Shot 2020-12-07 at 11 24 37 AM" src="https://user-images.githubusercontent.com/5618806/101395449-d07fc000-387e-11eb-9217-fc176da29bf5.png">

If you want to maintain this spacing, what about something like this?

```suggestion
* feat: implemented sanitize_feature_names specification +
  Allows users to configure a list of wildcard patterns to _remove_ items
  from the agent's HTTP header and `application/x-www-form-urlencoded` payloads.
  ** https://github.com/elastic/apm/blob/master/specs/agents/sanitization.md[spec]
  ** https://github.com/elastic/apm-agent-nodejs/blob/master/docs/configuration.asciidoc#sanitize-field-names[docs]
```

<img width="658" alt="Screen Shot 2020-12-07 at 11 28 37 AM" src="https://user-images.githubusercontent.com/5618806/101395921-703d4e00-387f-11eb-93b2-804abfa3b4b0.png">

Reply:
Ah, good to know.  My heart's still letting go of markdown :). Formatting changed to match the suggestion.